src/app/organisations/data/organisationsStorage.js

```diff
@@ -288,6 +283,46 @@ const getOrganisationStates = async () => {
   return Promise.resolve(categories);
 };
 
+const getUsersAssociatedWithOrganisationForApproval = async (userId) => {
```

Comment:
I found this method name a little confusing. The code does not do what the name suggests. Something like `getUsersPendingApprovalByUser` may be less ambigious?